providers/openlineage/src/airflow/providers/openlineage/plugins/listener.py

```diff
@@ -504,11 +502,12 @@ def on_state_change():
             adapter_kwargs = {
                 "run_id": task_uuid,
                 "job_name": get_job_name(ti),
-                "parent_job_name": dagrun.dag_id,
-                "parent_run_id": parent_run_id,
                 "end_time": end_date.isoformat(),
                 "task": OperatorLineage(),
-                "run_facets": get_airflow_debug_facet(),
+                "run_facets": {
+                    **get_task_parent_run_id(ti.dag_id, parent_run_id),
```

Comment:
```suggestion
                    **get_task_parent_run_id(parent_run_id=parent_run_id, parent_job_name=ti.dag_id),
```
I think the order here was wrong, so it would not work. Maybe some uuid validation should be added to catch those kind of errors or we should only allow kwargs?

Reply:
There is validation, I just did not run it with manual method. Added calls to the test now.